sample/src/main/java/com/polidea/rxandroidble2/sample/example2_connection/ConnectionExampleActivity.java

```diff
@@ -122,4 +128,12 @@ private void updateUI() {
         connectButton.setText(connected ? R.string.disconnect : R.string.connect);
         autoConnectToggleSwitch.setEnabled(!connected);
     }
+
+    @Override
+    protected void onPause() {
+        super.onPause();
+
+        triggerDisconnect();
+        compositeDisposable.dispose();
```

Comment:
```suggestion
        compositeDisposable.clear();
```
Otherwise the next time the activity will resume nothing will work